internal/helper/helper.go

```diff
@@ -284,12 +277,47 @@ func SafeQuantitySubtraction(orig, subtract quantity.Size) quantity.Size {
 	return orig - subtract
 }
 
+// CreateTarArchive places all of the files from a source directory into a tar.
+// Currently supported are uncompressed tar archives and the following
+// compression types: zip, gzip, xz bzip2, zstd
+func CreateTarArchive(src, dest, compression string, verbose, debug bool) error {
```

Comment:
This might be because we initially said 'just .tar' might be enough, but I don't see the `compression` argument used. Do we want to switch it on in a separate PR?

Reply:
Yes we agreed to do this as a separate task in a future pulse to keep this PR from getting even bigger.